<p>Referencing 
  <span data-role="tag" data-value="15e426f0-b95e-2151-5595-f726c176577b" id="idbc004e31-1726-da5b-bf64-e1b536d35266" data-type="figure">Figure 1</span>, a&nbsp;
  <span data-role="tag" data-value="2a778c08-37eb-4993-2a7f-e589e60034a6" id="id9a0ad4ae-00e2-44df-734d-a33d3d45474b" data-type="drawingObject">reaction system 100</span>&nbsp;illustrates a set of 
  <span data-role="tag" data-value="cde5d9f8-fc5f-2a41-992c-12e788e05193" id="idde3341ef-b751-7b73-b5e6-c88cae4f06ce" data-type="drawingObject">initial conditions and quantities 124</span>&nbsp;for a quantitiative Polymerase Chain Reaction that includes 
  <span data-role="tag" data-value="099bd707-6a25-f4e8-cc3b-c60fdbf55fd1" id="id87495cec-d6a7-485c-38cb-14c547f00f6c" data-type="drawingObject">reagent(s) 106</span>&nbsp;(e.g., polymerase, primers, probes, etc.,)&nbsp; and a&nbsp;
  <span data-role="tag" data-value="524bc823-97ae-7507-d6cc-96e7446f2d49" id="idb18ac38f-7481-8820-bcef-255e7e91fcde" data-type="drawingObject">sample 118</span>&nbsp;(e.g., target DNA strand, template DNA strand, etc.,). In qPCR, the 
  <span data-role="tag" data-value="524bc823-97ae-7507-d6cc-96e7446f2d49" id="id6b62e014-6615-0c73-0755-81d409dc92f5" data-type="drawingObject">sample 118</span>&nbsp;may contain DNA strands that serves as a template during the amplication process. The 
  <span data-role="tag" data-value="524bc823-97ae-7507-d6cc-96e7446f2d49" id="id8c8603a2-14c6-5288-7b7e-7ed8955c0e1d" data-type="drawingObject">sample 118</span>&nbsp;may under go through a sample preperation process prior to being combined with the 
  <span data-role="tag" data-value="099bd707-6a25-f4e8-cc3b-c60fdbf55fd1" id="id6e304926-fb19-2ae8-f3d9-6683f9bd70d2" data-type="drawingObject">reagent(s) 106</span>.
</p>
<p>In qPCR, the&nbsp;&nbsp;
  <span data-role="tag" data-value="cde5d9f8-fc5f-2a41-992c-12e788e05193" id="id5a5d08a7-3c5d-1e74-d65b-31e7fde8c96e" data-type="drawingObject">initial conditions and quantities 124</span>&nbsp;may additionally include&nbsp;
  <span data-role="tag" data-value="26bdcf06-8f53-3894-d6e6-c95b00f0b6a8" id="idef0159cb-2ef0-a8dc-cd60-2cb71104c6e9" data-type="drawingObject">quantities 122</span>&nbsp;for the&nbsp;
  <span data-role="tag" data-value="099bd707-6a25-f4e8-cc3b-c60fdbf55fd1" id="idc74f8ba3-5a5a-f220-55a6-fb2d8237e193" data-type="drawingObject">reagent(s) 106</span>&nbsp;and the 
  <span data-role="tag" data-value="524bc823-97ae-7507-d6cc-96e7446f2d49" id="ided44a67e-cc4e-05b5-652d-97c2652ff3f2" data-type="drawingObject">sample 118</span>, as well as supplemenatal information such as the&nbsp;location (e.g., reaction well, plate position, etc.,) where the 
  <span data-role="tag" data-value="099bd707-6a25-f4e8-cc3b-c60fdbf55fd1" id="id9b30b5b9-ea12-9b50-5811-9e1cdd560a51" data-type="drawingObject">reagent(s) 106</span>&nbsp;and the 
  <span data-role="tag" data-value="524bc823-97ae-7507-d6cc-96e7446f2d49" id="idb623e182-5db5-df27-3839-cc35bbf08d1e" data-type="drawingObject">sample 118</span>&nbsp;where placed&nbsp;in a 
  <span data-role="tag" data-value="58c3f7b8-f06d-773e-ec53-cafeecfe1002" id="id0526af10-f15e-883c-b2c3-aaa107639088" data-type="drawingObject">reaction vessel 102</span>&nbsp;(
  <span data-role="tag" data-value="929dc453-31e6-522a-9a50-5d1f077a62ea" id="id361795d0-6485-59b9-a3e9-4ba68f06a2ac" data-type="drawingObject">reaction site 116</span>). Environmental conditions may also factor into as part of the&nbsp;
  <span data-role="tag" data-value="cde5d9f8-fc5f-2a41-992c-12e788e05193" id="id8855ffe1-a3f2-a258-0b01-e2a059f395a1" data-type="drawingObject">initial conditions and quantities 124</span>&nbsp;including the 
  <span data-role="tag" data-value="6e38c37e-0d17-b362-ba94-b0cf53b90c81" id="idd078b4ad-cb64-9f3d-f09a-52c9da317339" data-type="drawingObject">temperature 108</span>&nbsp;and 
  <span data-role="tag" data-value="e732092a-a3f2-7790-c53a-ea22cce302bb" id="id2becbd83-24d9-e75a-7b8d-6d024642d701" data-type="drawingObject">pressure 120</span>&nbsp;at the start of and during the course of the qPCR reaction as changes in 
  <span data-role="tag" data-value="6e38c37e-0d17-b362-ba94-b0cf53b90c81" id="idbe0de14d-7bd6-7bc3-57bc-fff732455266" data-type="drawingObject">temperature 108</span>&nbsp;and 
  <span data-role="tag" data-value="e732092a-a3f2-7790-c53a-ea22cce302bb" id="id389623d5-64f1-5c66-b66c-17670729f1ba" data-type="drawingObject">pressure 120</span>&nbsp;may affect volumetric measurements.&nbsp;
</p>
<p>When the 
  <span data-role="tag" data-value="58c3f7b8-f06d-773e-ec53-cafeecfe1002" id="idc0fec37e-633e-57e4-eddd-0e0bbb3728c2" data-type="drawingObject">reaction vessel 102</span>&nbsp;is provided the 
  <span data-role="tag" data-value="50d794a1-910b-2a38-93e5-b65eb45502a8" id="idbe494bec-094f-8962-1b28-57a97ed6548f" data-type="drawingObject">instrument 104</span>&nbsp;to start the qPCR reaction, the 
  <span data-role="tag" data-value="cde5d9f8-fc5f-2a41-992c-12e788e05193" id="id7b98fc16-8e06-abd2-5c41-9e2422908c32" data-type="drawingObject">initial conditions and quantities 124</span>&nbsp;may be entered in or detcted by the 
  <span data-role="tag" data-value="50d794a1-910b-2a38-93e5-b65eb45502a8" id="idfcbf8762-ab50-24fc-afab-0285d3858cc6" data-type="drawingObject">instrument 104</span>&nbsp;and reported to an 
  <span data-role="tag" data-value="59aacb1b-2969-1143-91fe-3b352298d886" id="id6621e70b-60f5-3741-a445-2f9ee76666ef" data-type="drawingObject">initial reaction condition database 110</span>.&nbsp;
</p>
<p>During the reaction, the 
  <span data-role="tag" data-value="524bc823-97ae-7507-d6cc-96e7446f2d49" id="idf41def8e-d95f-c175-de70-b2f752e013d0" data-type="drawingObject">sample 118</span>&nbsp;is denatured during a high temperature phase of the reaction, separating the double stranded DNA in two complementary strands. High-temperature incubation is used to “melt” the double stranded DNA &nbsp;into single strands and loosen the secondary structure in single-stranded DNA. The highest temperature that the DNA polymerase can withstand is typically used (usually 95C). The denaturation time can be increased if template guanine cytosine(GC) content is high.
</p>
<p>An annealing phase follows the denaturing phase. During the annealing phase, complementary sequences have an opportunity to hybridize, so an appropriate temperature is used that is based on the calculated melting temperature (Tm) of the primers (typically this temperature is 5C&nbsp;below the Tm of the primer). During the annealing phase the primers and probes anneal to the&nbsp;single&nbsp;stranded DNA. The primers and probes anneal to specific complementary sequences of the single stranded DNA&nbsp;on either of the signle strands. &nbsp;The&nbsp;primers attach to specific sites&nbsp;of the DNA identifying a start location for&nbsp;the polymerase, the probes anneal to a site downstream of the primers. &nbsp;The probes may be utilized to identify a marker (e.g.,&nbsp;gene, phenotype, microsatellite sequence, SNP) of interest&nbsp;</p>
<p>Following the annealling phase, the reaction undergoes an extension/replication phase where the single strands of DNA are replicated. The extension/replication phase changes adjusts the temperature to&nbsp;70–72C, as this is where the activity of the DNA polymerase is optimal, and primer extension occurs at rates of up to 100 bases per second. When an amplicon in real-time PCR is small, this step is often combined with the annealing step, using 60C as the temperature. During replication/extension phase, the&nbsp;primers indicate an attachment point for the polymerase to begin extending the single stranded DNA&nbsp;of nucleotides adjacent to the primer nucleotides to the template DNA forming a complementary sequence and releasing the fluorescent dyes/tag when the probes are cleaved by the polymerase.&nbsp;</p>
<p>During the qPCR reaction, the 
  <span data-role="tag" data-value="50d794a1-910b-2a38-93e5-b65eb45502a8" id="id16cba30d-b803-6c5c-ace9-1b9903b83260" data-type="drawingObject">instrument 104</span>&nbsp;may detect the fluorescent emissions for the fluorescent probes. The fluorescent emissions may correspond to the&nbsp;intensity of emitted light (fluorescence) as a function of the wavelength of the emitted light used to identify specific probes. The 
  <span data-role="tag" data-value="50d794a1-910b-2a38-93e5-b65eb45502a8" id="id5c1bd2f4-afc1-8d31-fdb4-cdaa6660d34e" data-type="drawingObject">instrument 104</span>&nbsp;records these emssion or lackthereof as the results that of qPCR reaction and record this information in a&nbsp;
  <span data-role="tag" data-value="4f2e2d91-606b-469e-baa5-3e1dc7dabd1b" id="id8fa786db-b1da-5388-06a4-55b3574994b5" data-type="drawingObject">reaction results database 112</span>.
</p>
<p>Indentifying optimal reactants and reactant conditions is important in improving the reaction efficiency and subsequently the accuracy of a real time (rt) PCR data.&nbsp;</p>
<p>In a perfect scenario, each target copy in a PCR reaction will be copied at each cycle, doubling the number of full-length target molecules: this corresponds to 100% amplification efficiency. Variations in efficiency will be amplified as thermal cycling progresses. Thus, any deviation from 100% efficiency can result in potentially erroneous data.</p>
<p>One way to minimize efficiency bias is to amplify relatively short targets. Amplifying a 100 basepair (bp)&nbsp;region is much more likely to result in complete synthesis in a given cycle than, say, amplifying a 1,200 bp target. For this reason, real-time PCR target lengths are generally 60–200 bp. In addition, shorter amplicons are less affected by variations in template integrity. If nucleic acid samples are slightly degraded and the target sequence is long, upstream and downstream primers will be less likely to find their complementary sequence in the same DNA fragment.</p>
<p>Amplicon GC content and secondary structure can be another cause of data inaccuracy. Less-than-perfect target doubling at each cycle is more likely to occur if secondary structure obstructs the path of the DNA polymerase. Ideally, primers should be designed to anneal with, and to amplify, a region of medium (50%) GC content with no significant GC stretches. For amplifying cDNA, it is best to locate amplicons near the 3ʹ ends of transcripts. If RNA secondary structure prohibits full-length cDNA synthesis in a percentage of the transcripts, these amplicons are less likely to be impacted.</p>
<p>Target specificity is another important factor in data accuracy. When designing real-time PCR primers, check primers to be sure that their binding sites are unique in the genome. This reduces the possibility that the primers could amplify similar sequences elsewhere in the sample genome. Primer design software programs automate the process of screening target sequences against the originating genome and masking homologous areas, thus eliminating primer designs in these locations.</p>
<p>Genomic DNA(gDNA), pseudogenes, and allelic variants needed to be factored into consideration when considering different primer and amplicon designs.&nbsp;</p>
<p>gDNA carryover in an RNA sample may be a concern when measuring gene expression levels. The gDNA may be co-amplified with the target transcripts of interest, resulting in invalid data. gDNA contamination is detected by setting up control reactions that do not contain reverse transcriptase (no-RT controls); if the Ct for the no-RT control is higher than the Ct generated by the most dilute target, it indicates that gDNA is not contributing to signal generation. However, gDNA can compromise the efficiency of the reaction due to competition for reaction components such as dNTPs and primers.</p>
<p>The best method for avoiding gDNA interference in realtime PCR is thoughtful primer (or primer/probe) design, which takes advantage of the introns present in gDNA that are absent in mRNA. Whenever possible, Applied Biosystems™ TaqMan™ Gene Expression Assays are designed so that the TaqMan probe spans an exonexon boundary. Primer sets for SYBR Green dye–based detection should be designed to anneal in adjacent exons or with one of the primers spanning an exon/exon junction. When upstream and downstream PCR primers anneal within the same exon, they can amplify target from both DNA and RNA. Conversely, when primers anneal in adjacent exons, only cDNA will be amplified in most cases, because the amplicon from gDNA would include intron sequence, resulting in an amplicon that is too long to amplify efficiently in the conditions used for real-time PCR.</p>
<p>Pseudogenes, or silent genes, are other transcript variants to consider when designing primers. These are derivatives of existing genes that have become nonfunctional due to mutations and/or rearrangements in the promoter or gene itself. Primer design software programs can perform BLAST™ searches to avoid pseudogenes and their mRNA products.</p>
<p>Allelic variants are two or more unique forms of a gene that occupy the same chromosomal locus. Transcripts originating from these variants can vary by one or more mutations. Allelic variants should be considered when designing primers, depending on whether one or more variants are being studied. In addition, GC-content differences between variants may alter amplification efficiencies and generate separate peaks on a melt curve, which can be incorrectly diagnosed as off-target amplification. Alternately spliced variants should also be considered when designing primers.</p>
<p>Specificity, dimerization, and self-folding in primers and probes are another set of conditions that needed to be accounted for when considering different designs of a primers and amplicons.</p>
<p>Primer-dimers are most often caused by an interaction between forward and reverse primers, but can also be the result of forward-forward or reverse-reverse primer annealing, or a single primer folding upon itself. Primerdimers are of greater concern in more complex reactions such as multiplex real-time PCR. If the dimerization occurs in a staggered manner, as often is the case, some extension can occur, resulting in products that approach the size of the intended amplicon and become more abundant as cycling progresses. Typically, the lower the amount of target at the start of the PCR, the more likely primer-dimer formation will be. The positive side of this potential problem is that the interaction of primer-dimers is usually less favorable than the intended primer-template interaction, and there are many ways to minimize or eliminate this phenomenon.</p>
<p>The main concern with primer-dimers is that they may cause false-positive results. This is of particular concern with reactions that use DNA-binding dyes such as SYBR Green I dye. Another problem is that the resulting competition for reaction components can contribute to a reaction efficiency outside the desirable range of 90–110%. The last major concern, also related to efficiency, is that the dynamic range of the reaction may shrink, impacting reaction sensitivity. Even if signal is not generated from the primer-dimers themselves (as is the case with TaqMan Assays), efficiency and dynamic range may still be affected.</p>
<p>Several free software programs are available to analyze real-time PCR primer designs and determine if they will be prone to dimerize or fold upon themselves. The AutoDimer software program (authored by P.M. Vallone, National Institute of Standards and Technology, USA) is a bioinformatics tool that can analyze a full list of primers at the same time. This is especially helpful with multiplexing applications. However, while bioinformatics analysis of primer sequences can greatly minimize the risk of dimer formation, it is still necessary to monitor dimerization experimentally.</p>
<p>The traditional method of screening for primer-dimers is gel electrophoresis. Primer-dimers appear as diffuse, smudgy bands near the bottom of the gel. One concern with gel validation is that it is not very sensitive and therefore may be inconclusive. However, gel analysis is useful for validating data obtained from a melting/ dissociation curve, which is considered the best method for detecting primer-dimers.</p>
<p>Melting or dissociation curves should be generated following any real-time PCR run that uses DNA-binding dyes for detection. In brief, the instrument ramps from low temperature, in which DNA is double-stranded and fluorescence is high, to high temperature, which denatures DNA and results in lower fluorescence. A sharp decrease in fluorescence will be observed at the Tm for each product generated during the PCR. The melting curve peak obtained for the no-template control can be compared to the peak obtained from the target to determine whether primer-dimers are present in the reaction.</p>
<p>Ideally, a single distinct peak should be observed for each reaction containing template, and no peaks should be present in the no-template controls. Smaller, broader peaks at a lower melting temperature than that of the desired amplicon and also appearing in the no-template control reactions are quite often dimers. Again, gel runs of product can often validate the size of the product corresponding to the melting peak.</p>
<p>There are situations in which primer-dimers are present, but they may not affect the overall accuracy of the realtime PCR assay. A common observation is that primerdimers are present in the no-template control but do not appear in reactions containing template DNA. This is not surprising because in the absence of template, primers are much more likely to interact with each other. When template is present, primer-dimer formation is not favored.</p>
<p>As long as the peak seen in the no-template control is absent in the plus-template dissociation curve, primerdimers are not an issue.</p>
<p>Primer-dimers are part of a broad category of nonspecific PCR products that includes amplicons created when a primer anneals to an unexpected location with an imperfect match. Amplification of nonspecific products is of concern because they can contribute to fluorescence, which in turn artificially shifts the Ct of the reaction. They can influence reaction efficiency through competition for reaction components, resulting in a decreased dynamic range and decreased data accuracy. Nonspecific products are an even greater concern in absolute quantification assays, in which precise copy numbers are reported.</p>
<p>Standard gel electrophoresis is generally the first step in any analysis of real-time PCR specificity. While it can help to identify products that differ in size from a target amplicon, a band may still mask similar-sized amplicons and have limited sensitivity. Due to its accuracy and sensitivity, melting curve analysis provides the most confidence in confirming gel electrophoretic assessment of primer specificity.</p>
<p>While nonspecific amplification should always be eliminated when possible, there are some cases in which the presence of these secondary products is not a major concern. For example, if alternate isoforms or multiple alleles that differ in GC content are knowingly targeted, multiple products are expected.</p>
<p>When considering the design of certain Primers, the following following software options may be useful such as Applied Biosystems™ Primer Express™ Software, Invitrogen™ OligoPerfect™ Designer web-based tool, and Invitrogen™ Vector NTI™ Software.&nbsp;</p>
<p>These programs can automatically design primers for specific genes or target sequences using algorithms that incorporate the following guidelines and can also perform genome-wide BLAST searches for known sequence homologies.</p>
<blockquote>
  <p>• In general, design primers that are 18–28 nucleotides in length</p>
</blockquote>
<blockquote>
  <p>• Avoid stretches of repeated nucleotides</p>
</blockquote>
<blockquote>
  <p>• Aim for 50% GC content, which helps to prevent mismatch stabilization</p>
</blockquote>
<blockquote>
  <p>• Choose primers that have compatible Tm values (within 1°C of each other)</p>
</blockquote>
<blockquote>
  <p>• Avoid sequence complementarity between all primers employed in an assay and within each primer</p>
</blockquote>
<p>These considerations may be important to improve the effiiency of the system but may require the additional analysis of the&nbsp;
  <span data-role="tag" data-value="cde5d9f8-fc5f-2a41-992c-12e788e05193" id="id0c0bdec6-e470-2d83-fd48-618a6c2a4cb5" data-type="drawingObject">initial conditions and quantities 124</span>&nbsp;in comparison with the reaction 
  <span data-role="tag" data-value="9e8c6e61-8aa8-d838-8020-25fe7713b571" id="idf9b9c375-5fd3-2906-61e1-026bbcec17e8" data-type="drawingObject">results 114</span>&nbsp;of a plurality of similar reaction sets to identify and predict possible changes to improve the efficiency of other PCR reactions.&nbsp;
</p>